metals/src/main/scala/scala/meta/internal/metals/ForwardingMetalsBuildClient.scala

```diff
@@ -37,30 +50,36 @@ final class ForwardingMetalsBuildClient(
     clientConfig: ClientConfiguration,
     statusBar: StatusBar,
     time: Time,
-    didCompile: CompileReport => Unit,
-    onBuildTargetDidCompile: BuildTargetIdentifier => Unit,
+    didCompile: b.CompileReport => Unit,
+    onBuildTargetDidCompile: b.BuildTargetIdentifier => Unit,
     onBuildTargetDidChangeFunc: b.DidChangeBuildTarget => Unit,
 ) extends MetalsBuildClient
     with Cancelable {
 
+  private val forwarders = ListBuffer.empty[LogForwarder]
```

Comment:
should we wrap forwarders into atomic reference to be 100% safe because of multithreading?